platforms/ide/ide/src/main/java/org/gradle/plugins/ide/internal/tooling/model/DefaultGradleProject.java

```diff
@@ -38,7 +38,6 @@ public class DefaultGradleProject implements Serializable, GradleProjectIdentity
     private DefaultProjectIdentifier projectIdentifier;
     private DefaultGradleProject parent;
     private List<? extends DefaultGradleProject> children = new LinkedList<>();
-    private String buildTreePath;
```

Comment:
ℹ️  The build tree path is not exposed on the actual `GradleProject` TAPI model, so there is no need to send it as a part of the serialized object.